### ISSUE TITLE
stretch_both sizing mode doesn't stretch in 0.13

### ISSUE THREAD

**Issue by mrocklin** [bokeh/bokeh#8115]
When I upgraded to 0.13 I found that bokeh server plots that were rendered with "stretch_both" mode stretched somewhat oddly.  

![image](https://user-images.githubusercontent.com/306380/43358114-69877c48-9241-11e8-8e4d-990ca03b59f8.png)

When I resize the screen they continuously get bigger

![image](https://user-images.githubusercontent.com/306380/43358123-81037232-9241-11e8-82be-4fe8cc79ace1.png)

I've found this to be true on both chrome and firefox.  The problem does not occur on 0.12.16 .  My apologies for not including a minimal example.  My hope is that the release window is small enough that something here will trigger someone's memory.  I reproduce this with dask by running 

    dask-scheduler

and then navigating to http://localhost:8787/tasks

**Comment by mattpap** [comment]
The algorithm for computing available space in the DOM changed, so most likely this is the reason. To say more, I will need to check the HTML that interacts with bokeh. However, I'm redoing the layout from scratch for 1.0, so hopefully all those layout related issues will go away.

**Comment by birdsarah** [comment]
Just to confirm that as a dask user this is definitely an issue.

**Comment by bryevdv** [comment]
@mrocklin I suspect this is fixed in 1.1dev builds after #8085. At least, the `stretch_both` examples I have tested manually all work as expected, and there also some new CI tests for these as well. Since there is not really any MRE here I am going to go ahead and close this. If there is still a demonstrable, reproducible issue, please re-open with a test case. (or open a new issue if that is more appropriate)
